docs/integrations/sources/googleanalytics.md

```diff
@@ -1,44 +1,37 @@
 # Google Analytics
 
-## Overview
-
-The Google Analytics source supports Full Refresh syncs. That is, every time a sync is run, Airbyte will copy all rows in the tables and columns you set up for replication into the destination in a new table.
-
-This Google Analytics source wraps the [Pipelinewise Singer Google Analytics Tap](https://github.com/transferwise/pipelinewise-tap-google-analytics).
-
-### Output streams
-
-* `website_overview`
-* `traffic_sources`
-* `pages`
-* `locations`
-* `monthly_active_users`
-* `four_weekly_active_users`
-* `two_weekly_active_users`
-* `weekly_active_users`
-* `daily_active_users`
-* `devices`
-* Any custom reports you configure. See the section below on custom reports. 
-
-Please reach out to us on Slack or [create an issue](https://github.com/airbytehq/airbyte/issues) if you need to send custom Google Analytics report data with Airbyte.
-
-### Features
+## Features
 
 | Feature | Supported? |
 | :--- | :--- |
 | Full Refresh Sync | Yes |
 | Incremental Sync | Yes |
 | Replicate Incremental Deletes | No |
 | SSL connection | Yes |
-| Namespaces | No |
+| Custom Reports | Yes |
 
-### Performance considerations
+This Google Analytics source wraps the [Pipelinewise Singer Google Analytics Tap](https://github.com/transferwise/pipelinewise-tap-google-analytics).
 
-The Google Analytics connector should not run into Google Analytics API limitations under normal usage. Please [create an issue](https://github.com/airbytehq/airbyte/issues) if you see any rate limit issues that are not automatically retried successfully.
+## Supported Tables
+
+* website_overview
```

Comment:
```suggestion
* `website_overview`
```

Reply:
Removing the whole page, so this will be unnecessary!